### ISSUE TITLE
Remove unused code for Proxy Domain Registrar

### ISSUE THREAD

**Issue by siman** [dappforce/subsocial-parachain#160]

**Comment by siman** [comment]
Need to delete this code:
```
ProxyType::DomainRegistrar => matches!(
	c,
	Call::Domains(pallet_domains::Call::force_register_domain { .. })
),
```

**Comment by bolajahmad** [comment]
The code here is an implementation of the trait `pub trait InstanceFilter<T>: Sized + Send + Sync {`.

We do not need the implementation function, how are we sure that it's not used elsewhere? Are there some tests I could run to verify that it works still other than building the binary

**Comment by F3Joule** [comment]
Closed by #163
